modules/base-laws/src/test/scala/io/janstenpickle/trace4cats/base/context/laws/IdTests.scala

```diff
@@ -0,0 +1,16 @@
+package io.janstenpickle.trace4cats.base.context.laws
+
+import cats.~>
+import cats.laws.discipline.SerializableTests
+import io.janstenpickle.trace4cats.base.context.Unlift
+import io.janstenpickle.trace4cats.base.context.laws.discipline.UnliftTests
+import org.scalacheck.Cogen
+
+class IdTests extends BaseSuite {
+  implicit val cogenOption2Option: Cogen[Option ~> Option] =
+    Cogen(k => k(Some(0)).size.toLong)
+
+  checkAll("Option <~> Option", UnliftTests[Option, Option].unlift[String, Int])
```

Comment:
@mrdziuban UnliftTests extends LiftTests, so we need only 1 file and 1 test invokation, not two.